packages/rest/src/router/routing-table.ts

```diff
@@ -238,6 +240,31 @@ export interface ResolvedRoute extends RouteEntry {
   readonly schemas: SchemasObject;
 }
 
+export class StaticRoute implements RouteEntry {
+  public readonly verb: string = 'get';
+  public readonly path: string = '*';
+  public readonly spec: OperationObject = {responses: {}};
+
+  constructor(private _handler: Function) {}
+
+  updateBindings(requestContext: Context): void {
+    // no-op
+  }
+
+  async invokeHandler(
+    requestContext: Context,
+    args: OperationArgs,
+  ): Promise<OperationRetval> {
+    const req = await requestContext.get(RestBindings.Http.REQUEST);
+    const res = await requestContext.get(RestBindings.Http.RESPONSE);
+    return this._handler(req, res);
```

Comment:
I am proposing to simplify the design here. Since there will be always at most one StaticRoute instance registered with a server, there is no need to make the handler configurable. Let's move the code dealing with `_routerForStaticAssets` here into `invokeHandler`.

Ideally, I'd like to see `_routerForStaticAssets` to be a member property of `StaticRoute`.